guides/README.md

```diff
@@ -80,14 +91,16 @@ Never write "Foreman" or "Satellite" words directly but use the following variab
 
 The table only covers the most frequent terms, the rest is defined in the [included file](common/attributes.adoc).
 
-Variables cannot be used in shell or code examples. To use them, use "attributes" keyword:
+Variables cannot be used in shell or code examples.
+To use them, use "attributes" keyword:
 
 	[options="nowrap" subs="+quotes,attributes"]
 	----
 	# ls {VariableName}
 	----
 
-Hide or show specific blocks, paragraphs, warnings or chapters via special variable called "build". Its value can be set either to "foreman" or "satellite":
+Hide or show specific blocks, paragraphs, warnings or chapters via special variable called "build".
+Its value can be set either to "foreman" or "satellite":
```

Comment:
This is a markdown file, though the one-sentence-per-line format applies here too.